network/network.go

```diff
@@ -244,3 +244,25 @@ func (nm *networkManager) getNetwork(networkId string) (*network, error) {
 
 	return nil, errNetworkNotFound
 }
+
+// getNetworkNameForNetNs finds the network that contains the endpoint that was created for this netNs. Returns
+// and errNetworkNotFound if the netNs is not found in any network
+func (nm *networkManager) FindNetworkNameFromNetNs(netNs string) (string, error) {
```

Comment:
FindNetworkIDFromNetNs?

Reply:
Same, also thinking we should put a not on what the param is